migrations/20160106061843_fix_zero_dates.php

```diff
@@ -19,7 +19,7 @@ class FixZeroDates extends AbstractMigration
 
     public function up()
     {
-        $this->execute("SET SESSION sql_mode = ''");
+        //$this->execute("SET SESSION sql_mode = ''");
```

Comment:
These changes were put in place [because of this issue](https://github.com/opencfp/opencfp/issues/395). If we support MySQL versions < 5.7.8, we'll need some special sauce for this. From your comment in that issue, I also don't like the idea of having to change `mysqld` settings to get the migrations to run so this might cause a regression, but I don't have an old version of MySQL to test on *right now*.

Reply:
Same thing with this one.

There is no big urgency on this PR but it is laying the groundwork for allowing the app to be deployed to Heroku